### PR TITLE
updated all links to old docs

### PR DESCRIPTION
**Pull request by shaynacohen**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5167420f7ff14447d98998b46b586ee5d2a6a0f3/index.html)

File: docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ You’ve now enabled a Set custom entity, created a Set entity called “Dining
...
 
-{% include info title="Note" content="It’s a good practice to test configurations in a *cloned* configuration, so as not to affect the active production configuration. The cloning process creates a *copy* of the configuration that you can edit safely before pushing changes to the live config. You can find details on cloning your config in the [Configuration Staging and Rollout](https://support.shotgunsoftware.com/hc/en-us/articles/219033168-Configuration-staging-and-rollout#Cloning%20your%20Configuration) document." %}
+{% include info title="Note" content="It’s a good practice to test configurations in a *cloned* configuration, so as not to affect the active production configuration. The cloning process creates a *copy* of the configuration that you can edit safely before pushing changes to the live config. You can find details on cloning your config in the [Configuration Staging and Rollout](https://developer.shotgridsoftware.com/60762324/?title=Configuration+Staging+and+Rollout#cloning-your-configuration) document." %}
```
Comment (drewavisautodesk) [inline docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md:131]: curly quote

File: docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md [hunk 2]
```diff
@@ -381,7 +381,7 @@ The **Work Area**: is displaying **.../{% include product %}/projects/the_other_
 
 ### Extending the example
 
-In this example, we modified a single template, but there’s plenty more you can do with your filesystem configuration. In a real world example, you’d likely change *all* asset-related files to have the same file naming conventions. You can make modifications based on other entities (Season, Episode, Level, etc.), create user folders, name your folders based on {% include product %} data manipulated with regular expressions, and much more. You can learn about all of Toolkit’s folder and schema options in the [Filesystem Configuration Reference](https://support.shotgunsoftware.com/hc/en-us/articles/219039868).
```
Comment (drewavisautodesk) [inline docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md:384]: Another couple of curly quotes.  I think maybe we should just do a separate PR to find and replace all of these, if they do indeed cause any kind of publishing issue (which it doesn't look like they do).

File: docs/en/guides/pipeline-integrations/getting-started/installing_app.md [hunk 3]
```diff
@@ -406,11 +406,11 @@ apps.tk-multi-pythonconsole.location:
   type: app_store
   name: tk-multi-pythonconsole
   version: v1.1.2
-```' %}
```
Comment (drewavisautodesk) [inline docs/en/guides/pipeline-integrations/getting-started/installing_app.md:409]: Is this deletion on purpose?  I looks like it matches the {% on line 381?  It might have to go on its own line.

File: docs/en/quick-answers/administering/where-is-my-cache.md [hunk 4]
```diff
@@ -60,6 +60,6 @@ If your configuration uses a shared core, then this will be located inside your
 
 ## Thumbnails
  
-Thumbnails used by Toolkit apps (like the [Loader](https://support.shotgunsoftware.com/entries/95442527)) are stored in the local Toolkit cache. They are stored per Project, Pipeline Configuration, and App (as needed). The structure beneath the root cache directory is as follows:
+Thumbnails used by Toolkit apps (like the [Loader](hhttps://developer.shotgridsoftware.com/a4c0a4f1/?title=Loader)) are stored in the local Toolkit cache. They are stored per Project, Pipeline Configuration, and App (as needed). The structure beneath the root cache directory is as follows:
```
Comment (drewavisautodesk) [inline docs/en/quick-answers/administering/where-is-my-cache.md:63]: hhttps probably wrong here, and there's an extra ) on the end of the URL

**Review by drewavisautodesk — COMMENTED** [review]
General comment - there are a lot of links with the "?title=" argument in them, but I don't think that's necessary.  EG: [Apps and Engines Configuration Reference](https://developer.shotgridsoftware.com/162eaa4b/?title=Pipeline+Integration+Components)
  Could be shortened to:
  [Apps and Engines Configuration Reference](https://developer.shotgridsoftware.com/162eaa4b/)
  
  There are some that include heading references, like: 
  [Passing Commandline Arguments](https://developer.shotgridsoftware.com/1b9c259a/?title=Launch+App#use-command-line-arguments-at-launch)    
Which can be:
[Passing Commandline Arguments](https://developer.shotgridsoftware.com/1b9c259a/#use-command-line-arguments-at-launch)    

I'm not sure if it's worth changing these, it's just to improve readability, and these docs are probably in maintenance mode.  Just an observation.

File: docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md (hunk 1 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md]: Thanks! Cleaned up a few of these I noticed lingering around.

File: docs/en/quick-answers/administering/where-is-my-cache.md (hunk 4 above)
Comment (shaynacohen) [reply docs/en/quick-answers/administering/where-is-my-cache.md]: Thanks! Spotted a couple more and cleaned these up as well.

File: docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md (hunk 2 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md:384]: I see {% include product %}, but I'm not seeing any additional curly quotes... can you let me know if that's right?

**Comment by shaynacohen** [comment]
@drewavisautodesk - thanks for reviewing and for your catches! I went ahead and fixed the arguments. There was one comment that I replied to as I could not find the issue. Can you review once more and let me know if it LGTM? Thanks!

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/02d64e5ef1498f5f0b956b91123bd5a75bc3e911/index.html)

File: docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md (hunk 2 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md:384]: Ah! Sorry - my brain is still coming online... curly quotes, not %}! Yes, I feel like this should be another PR to clean these up as well.

File: docs/en/guides/pipeline-integrations/getting-started/installing_app.md (hunk 3 above)
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/getting-started/installing_app.md:409]: This should be cleaned up now

**Review by drewavisautodesk — APPROVED** [review]
